<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AcroLap15" ignored="0" name="DemographicsTest" tests="3" failures="1" timestamp="05 Feb 2018 12:40:52 GMT" time="109.657" errors="0">
  <testcase name="DemographicsPageTest" time="77.675" classname="com.acams.ddf.testcases.DemographicsPageTest">
    <skipped/>
  </testcase> <!-- DemographicsPageTest -->
  <testcase name="DemographicsPageTest" time="24.849" classname="com.acams.ddf.testcases.DemographicsPageTest">
    <skipped/>
  </testcase> <!-- DemographicsPageTest -->
  <testcase name="DemographicsPageTest" time="6.276" classname="com.acams.ddf.testcases.DemographicsPageTest">
    <failure type="java.lang.AssertionError" message="test case t2 block got executed">
      <![CDATA[java.lang.AssertionError: test case t2 block got executed
	at com.acams.ddf.base.BaseTest.reportFailure(BaseTest.java:815)
	at com.acams.ddf.testcases.DemographicsPageTest.DemographicsPageTest(DemographicsPageTest.java:152)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- DemographicsPageTest -->
</testsuite> <!-- DemographicsTest -->
